Do-It/Controller/DueDatePickerTableViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  DueDatePickerTableViewController.swift
+//  Do-It
+//
+//  Created by Aaron on 2/21/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import UIKit
+
+protocol DueDatePickerTableViewControllerDelegate: AnyObject {
+    func dueDatePickerTableViewController(_ viewController: DueDatePickerTableViewController,
+                                          didSelectTimeRange range: (DateComponents, DateComponents)?)
+}
+
+class DueDatePickerTableViewController: UITableViewController {
+
+    var timeRangeSelected: (DateComponents, DateComponents)?
```

Comment:
One last change: name the components of the tuple for clearer readability:
`var timeRangeSelected: (rangeStartOffset: DateComponents, rangeEndOffset: DateComponents)?`